src/main/scala/amba/ahb/RegisterRouter.scala

```diff
@@ -115,6 +121,9 @@ trait HasAHBControlRegMap { this: RegisterRouter =>
     undefZero = undefZero,
     executable = executable)
 
+  // Backwards-compatibility default node accessor with no clock crossing
```

Comment:
should *these* be deprecated too?

Reply:
I think it is fine to keep them, it lets you connect to the device without having to import the clock crossing types just so that you can specify that you don't want to have one.